html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <form action="???">
+    <h1>Coding Form!</h1>
+    <div>
+      <label for="email">Email</label>
+    </div>
+    <input type="text" id="email">
+    <div>
+      <label for="option">Why is coding cool?</label>
+    </div>
+    <select id="option">
+      <option selected="selected">Select an option</option>
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+    </select>
+    <p><label for="favLanguage">Favorite Language (choose one)</label></p>
+
+    <input type="radio" name="language" id="favLanguage" value="HTML">HTML
```

Comment:
each of these inputs needs a label, you can wrap the input with the label if desired